src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -213,8 +213,18 @@ def _load_package_tool(tool_name, module_name, class_name, method_name, node_inp
                 message=f"Required inputs {list(missing_inputs)} are not provided for tool '{tool_name}'.",
                 target=ErrorTarget.EXECUTOR,
             )
+        try:
+            api = getattr(provider_class(**init_inputs_values), method_name)
```

Comment:
I remember we will deprecate BuiltinsManager. Do we need add similar logic in another position?
@thy09